src/build-data/cc/msvc.txt

```diff
@@ -48,7 +48,7 @@ default-debug -> "$(LINKER) /DLL /DEBUG"
 
 <binary_link_commands>
 default       -> "$(LINKER)"
-default-debug -> "$(LINKER) /DEBUG"
```

Comment:
Hmm, i don't know. Most people probably not. I'm debugging Botan most of the time directly in Visual Studio with a VS-Solution generated by Bakefile so i don't need the other PDB files.

An other solution to this problem is the one suggested by @agrandville in #432 by changing the filename of the CLI tool to botan-cli.exe. I'm happy with both ways.